ee/ee_controllers/middleware.go

```diff
@@ -0,0 +1,18 @@
+package ee_controllers
+
+import (
+	"github.com/gravitl/netmaker/logic"
+	"github.com/gravitl/netmaker/servercfg"
+	"net/http"
+	"strings"
+)
+
+func OnlyServerAPIWhenUnlicensedMiddleware(handler http.Handler) http.Handler {
+	return http.HandlerFunc(func(writer http.ResponseWriter, request *http.Request) {
+		if servercfg.ErrLicenseValidation != nil && !strings.HasPrefix(request.URL.Path, "/api/server") {
+			logic.ReturnErrorResponse(writer, request, logic.FormatError(servercfg.ErrLicenseValidation, "unauthorized"))
+			return
```

Comment:
the http response status should be forbidden rather than `unauthorized`

if you are limiting to only to allow status API the above check should be -> `if servercfg.ErrLicenseValidation != nil && request.URL.Path != /api/server/status`

Reply:
https://github.com/gravitl/netmaker/pull/2474/commits/ec6f6dbb7fa04ffaf8fb663409ec52262792c57e